datadog_checks_dev/datadog_checks/dev/tooling/commands/service_checks.py

```diff
@@ -0,0 +1,147 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+import json
+from collections import OrderedDict
+
+import click
+from six import string_types
+
+from .utils import CONTEXT_SETTINGS, abort, echo_failure, echo_info
+from ..constants import get_root
+from ...compat import JSONDecodeError
+from ...utils import file_exists, read_file
+
+REQUIRED_ATTRIBUTES = {
+    'agent_version',
+    'check',
+    'description',
+    'groups',
+    'integration',
+    'name',
+    'statuses',
+}
+
+
+def parse_version_parts(version):
+    return (
+        [int(v) for v in version.split('.') if v.isdigit()]
+        if isinstance(version, string_types) else []
+    )
+
+
+@click.group(
+    context_settings=CONTEXT_SETTINGS,
+    short_help='Manage service checks files'
+)
+def service_checks():
+    pass
+
+
+@service_checks.command(
+    context_settings=CONTEXT_SETTINGS,
+    short_help='Validate all `service_checks.json` files'
+)
+def verify():
+    """Validate all `service_checks.json` files."""
+
+    failed = 0
+    root = get_root()
+
+    for check_name in sorted(os.listdir(root)):
+        service_checks_file = os.path.join(root, check_name, 'service_checks.json')
+
+        if file_exists(service_checks_file):
+            display_queue = [(echo_info, '{} ->'.format(check_name))]
```

Comment:
not sure I like this strategy, there is value in printing what the check is doing, like:
```
Checking active_directory.... OK
Checking activemq... OK
Checking activemq_xml... FAILED
  Attribute `foo` is invalid
  Attribute `bar` is required
Checking agent_metrics... OK
...
```